devices/Scd30/Scd30Sensor.cs

```diff
@@ -0,0 +1,419 @@
+﻿// Copyright (c) 2017 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+
+using System;
+using System.Buffers.Binary;
+using System.IO.Ports;
+using Iot.Device.Scd30.Entities;
+using UnitsNet;
+
+namespace Iot.Device.Scd30
+{
+    /// <summary>
+    /// Sensirion SCD30 Sensor Module (CO2, humidity, and temperature sensor).
+    /// </summary>
+    /// <remarks>
+    /// Most of the SCD30 documentation below is derived or copied from the original documentation
+    /// at https://sensirion.com/media/documents/D7CEEF4A/6165372F/Sensirion_CO2_Sensors_SCD30_Interface_Description.pdf .
+    /// </remarks>
+    public class Scd30Sensor
```

Comment:
Since I don't new the SerialPort, shouldn't I leave the disposing or closing of it to the developer? They could be re-using the SerialPort for other purposes besides the SCD30.

Reply:
Fair enough regarding your other comment!